pyas2lib/utils.py

```diff
@@ -47,15 +48,15 @@ def _handle_application_pkcs7_mime(self, msg):
             self._fp.write(payload)
 
 
-def mime_to_bytes(msg, header_len):
+def mime_to_bytes(msg, policy):
```

Comment:
You have used the same name `policy` as in the import... this will not good please rename as email_policy

Reply:
OK